compiler/src/iree/compiler/Preprocessing/Common/test/pad_to_intrinsics_mfma.mlir

```diff
@@ -207,7 +207,8 @@ func.func @main2(%arg0: tensor<2x130x130x4xf16>, %arg1: tensor<3x3x4x320xf16>, %
 
 // -----
 
-// We want to skip padding skinny matmul cases, since warpReduction is more performant for it.
+// We want to skip padding skinny matmul cases, since WarpReduction is more performant for it.
+// TODO(newling) reconsider this, as WarpReduction is removed.
```

Comment:
we dont need to reconsider this, warp reduction is still a "strategy", just the llvmgpuvectordistribute pipeline can handle it.